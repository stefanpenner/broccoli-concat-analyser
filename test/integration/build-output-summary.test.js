'use strict';

const tmp = require('tmp');

const buildOutputSummary = require('../../lib/build-output-summary');

describe('build-output-summary', function() {
  let tmpPath;
  beforeEach(function() {
    tmpPath = tmp.dirSync().name;
  });
  it('generates final summary output', function() {
    let result = buildOutputSummary(tmpPath);
    expect(result).toMatchSnapshot();
  });
});

