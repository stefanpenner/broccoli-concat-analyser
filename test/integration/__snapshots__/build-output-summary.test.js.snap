// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build-output-summary generates final summary output 1`] = `
Object {
  "groups": Array [],
}
`;
